src/components/table/__tests__/TableView.selection-store.test.tsx

```
@@ -0,0 +1,41 @@
+import { render, screen, fireEvent } from '@testing-library/react';
```

ESLint

[error] 'screen' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)